src/PeakFinder.hpp

```diff
@@ -0,0 +1,86 @@
+#ifndef SHASTA_PEAKFINDER_HPP
+#define SHASTA_PEAKFINDER_HPP
+
+
+#include <experimental/filesystem>
+#include <stdexcept>
+#include <iostream>
+#include <utility>
+#include <vector>
+
+using std::experimental::filesystem::create_directories;
```

Comment:
If this is somehow needed: we should not use `experimental/filesystem` because it will create portability issues. Instead, if necessary use the functions defined in namespace `shasta/filesystem` (see `shasta/src/filesystem.hpp`). If the functionality you need is missing, add it there. But I looks like none of this is actually needed, like @bagashe points out.

Reply:
Agh this is a vestige of the implementation I wrote separately for testing, I will remove it.